README.md

```diff
@@ -69,6 +68,7 @@ A curated list of resources for learning about natural language processing, text
 ## Blog Articles, Papers, Case Studies
 
 ### General
+- [WEF Live Campaign - Twitter fed Global News Topics & Sentiment Tracker] (https://365.weflive.com/#!/r/worldwide/t/all-topics) - Live Jan 2019
```

Comment:
You have a space between the `[]` and `()` brackets, which causes these links to not be rendered correctly in the markdown. Please change them all to this format: 

```suggestion
- [WEF Live Campaign - Twitter fed Global News Topics & Sentiment Tracker](https://365.weflive.com/#!/r/worldwide/t/all-topics) - Live Jan 2019
```